apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -17,10 +18,25 @@ import './SupportDetailWip.css';
 
 function Header(props: { title: string }) {
   const { title } = props;
+  const navigate = useNavigate();
   return (
     <nav className="support-header">
-      <ArrowBackIosNewIcon />
-      <Typography variant="h4">{title}</Typography>
+      <IconButton
+        aria-label="back"
+        sx={{ color: 'text.primary' }}
```

Comment:
This key is wrong, so it's no-op (the icon is black). Perhaps you meant `sx={{ color: 'primary.900' }}`?

Reply:
No, it's on purpose. The color of the back chevron should match the primary text color. `primary.900` gives the color below:
![image](https://user-images.githubusercontent.com/29354437/226119338-6cfdf4b8-ad38-4fc9-9dcc-5061980ee5f6.png)

Reply:
Isn't that the colour we want? Am I misunderstanding something?